README.md

```diff
@@ -31,6 +29,42 @@ To run this application, you need to first clone the getting-started-with-the-do
+
+## Rating shapes
+
+The Rating control in .NET MAUI allows the use of both predefined shapes and custom shapes.
+
+## Set as read-only
+
+The .NET MAUI Rating control can also be used in a read-only state. In this mode, users cannot interact with the control.
+
+## Customization
+
+The .NET MAUI Rating control supports customizing the item color, item border color, spacing, selection color, and more, enabling the items to fit the application theme.
+
+## Related links
+[Learn More about .NET MAUI Rating](https://www.syncfusion.com/maui-controls/maui-rating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[Download Free Trial](https://www.syncfusion.com/downloads/maui/confirm?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[Pricing](https://www.syncfusion.com/sales/teamlicense?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[Documentation](https://help.syncfusion.com/maui/rating/getting-started?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[View Demos](https://github.com/syncfusion/maui-demos/tree/master/MAUI/Inputs/SampleBrowser.Maui.Inputs/Samples/Rating)
+[Community Forums](https://www.syncfusion.com/forums/maui?control=rating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/maui?control=sfrating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+[Online example](https://github.com/SyncfusionExamples/https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-rating-control?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples)
+
+## About Syncfusion .NET MAUI Rating Control
+The Syncfusion's [.NET MAUI Rating Control](https://www.syncfusion.com/maui-controls/maui-rating?utm_source=github&utm_medium=listing&utm_campaign=maui-rating-github-samples) library offers over 90 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to Speed Dial, we provide popular Blazor Components such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples), [Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-speed-dial-github-samples).
```

Comment:
Already explained about this, please consider this in all maui controls
![image](https://github.com/user-attachments/assets/ec2bf986-5010-4031-a4ea-4903be478b8d)